cropper/src/main/res/xml/provider_paths.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths>
+    <external-cache-path name="external_cached_cropper_files" path="." />
```

Comment:
* You're in root of app-specific cache directory. Work with library-specific directory in the cache directory instead.
* `name` should reflect what's inside the directory. This Uri is only shared with camera apps, so name it accordingly.
* `cropper_files` is implicit, it's the library's content provider. `external` is an implementation detail, Uri is meant to abstract the real storage.

    <external-cache-path name="camera" path="com.theartofdev.edmodo.cropper/camera" />